### PR TITLE
Use placemark's polyline in favor of mapbox's polyline

### PR DESCRIPTION
**Pull request by clementmas**
Fixes: https://github.com/maplibre/maplibre-gl-directions/issues/212

Let me know if I need to make other changes.

### CODE REVIEW

**Comment by smellyshovel** [comment]
@clementmas LGTM, but will have to check certain things manually, which I'm unable to do right now. Hopefully this is not urgent. Thanks for submitting this!

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The plugin is written 100% in TypeScript and therefore ships with built-in types
 ## Installation
 
 ```shell
-$ npm i @maplibre/maplibre-gl-directions
```
[the remark this answers is not in the document]
Comment (clementmas) [reply README.md:50]: The dollar sign breaks the copy/paste button:

```bash
$ npm i @maplibre/maplibre-gl-directions
'$' is not recognized as an internal or external command, operable program or batch file.
```

**Comment by clementmas** [comment]
Yes, no rush. Thanks for being open to this PR.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (smellyshovel) [reply README.md:50]: @clementmas Ah, Ok. It's just that it's usually used as an indication of the terminal prompt with no SU rights requested. But the copy-paste functionality is more important, I agree :+1:

**Review by smellyshovel — APPROVED** [review]

**Comment by smellyshovel** [comment]
Thanks for the commit, @clementmas :+1:

**Pull request merged**